src/CfgEditor/EdgeTPUCfgData.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ini from "ini";
+import { ICfgData } from "./ICfgData";
+
+const sections = ["edgetpu-compiler", "edgetpu-compile", "edgetpu-profile"];
+
+export class EdgeTpuCfgData implements ICfgData {
+  private _edgeTpuConfig: any = undefined;
+
+  constructor() {}
+
+  getAsConfig(): any {
+    return this._edgeTpuConfig;
+  }
+
+  getAsString(): string {
+    return ini.stringify(this._edgeTpuConfig);
+  }
+
+  setWithConfig(cfg: any): void {
+    this._edgeTpuConfig = cfg;
+    this.resolveDuplicated();
+  }
+
+  setWithString(text: string): void {
+    this._edgeTpuConfig = ini.parse(text);
+    this.resolveDuplicated();
+  }
+
+  private resolveDuplicated(): void {
+    // NOTE 'intermediate_tensors' and a set of 'search_delegate' and 'delegate_search_step' cannot be duplicated.
+    //      Therefore, when only 'intermediate_tensors' and a set of 'search_delegate' and 'delegate_search_step' is used together, those three options will be deleted.
```

Comment:
What is the case that user have both "intermediate_tesnsors" and "search_delegate"?
And what will happen? I think you are trying to avoid some kinda mal-functioning with this code?

Reply:
In our opinion, there is no case of using the two options together.
However, if people accidentally apply the two options and compile them, the following error occurs.
![image](https://github.com/Samsung/ONE-vscode/assets/43038815/5011750d-2794-4b1f-a523-b2564c380217)

As you said, if you want to set it up together to prevent this error, you chose to clear all of those options.

Reply:
However editing the file without user's consent... it sounds a bit unnatural. How about putting an error message for that case?

Reply:
We have already made a plan to strengthen the UI in the second way 😀

As you said, I will allow the user to do anything in the cfg file, and if there is a problem, I will correct it so that you can enter the appropriate error message when running :)